src/control/cmd/daos/container.go

```diff
@@ -0,0 +1,834 @@
+//
+// (C) Copyright 2021 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"unsafe"
+
+	"github.com/google/uuid"
+	"github.com/jessevdk/go-flags"
+	"github.com/pkg/errors"
+
+	"github.com/daos-stack/daos/src/control/drpc"
+	"github.com/daos-stack/daos/src/control/lib/txtfmt"
+	"github.com/daos-stack/daos/src/control/logging"
+)
+
+/*
+#include <daos.h>
+
+#include "daos_hdlr.h"
+*/
+import "C"
+
```

Comment:
ah, maybe that's what i was looking for. A comment here saying that this is the primary table with all cmd handler would help.